### PR TITLE
LPTOCPCI-47 Initial PR for OpenJDK with cluster creation

### PR DESCRIPTION
**Pull request by smatula**

### CODE REVIEW

File: ci-operator/config/rh-openjdk/xpaas-openjdk/rh-openjdk-xpaas-openjdk-main__openjdk-ocp4.13-lp-interop.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+releases:
+  initial:
```
Comment (ascerra) [inline ci-operator/config/rh-openjdk/xpaas-openjdk/rh-openjdk-xpaas-openjdk-main__openjdk-ocp4.13-lp-interop.yaml:11]: you can remove initial here we just use latest

File: ci-operator/config/rh-openjdk/xpaas-openjdk/OWNERS [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+# Fetched from https://github.com/rhpit/interop-tests root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- ascerra
+- madunn
+- minlei98
+- ryankwilliams
```
Comment (ryankwilliams) [inline ci-operator/config/rh-openjdk/xpaas-openjdk/OWNERS:11]:
```suggestion
```

File: ci-operator/config/rh-openjdk/xpaas-openjdk/OWNERS [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+- vi-patel
+options: {}
+reviewers:
+- ascerra
+- madunn
+- minlei98
+- ryankwilliams
```
Comment (ryankwilliams) [inline ci-operator/config/rh-openjdk/xpaas-openjdk/OWNERS:18]:
```suggestion
```

File: ci-operator/config/rh-openjdk/xpaas-openjdk/OWNERS [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (ryankwilliams) [inline ci-operator/config/rh-openjdk/xpaas-openjdk/OWNERS:1]: This comment can probably be removed, looks to be a copy/paste.

File: ci-operator/jobs/rh-openjdk/xpaas-openjdk/OWNERS [hunk 5]
```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT; this file is auto-generated using https://github.com/openshift/ci-tools.
```
Comment (ryankwilliams) [inline ci-operator/jobs/rh-openjdk/xpaas-openjdk/OWNERS:1]: This comment can probably be removed, looks to be a copy/paste.

File: ci-operator/jobs/rh-openjdk/xpaas-openjdk/OWNERS [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+# Fetched from https://github.com/rhpit/interop-tests root OWNERS
+# If the repo had OWNERS_ALIASES then the aliases were expanded
+# Logins who are not members of 'openshift' organization were filtered out
+# See the OWNERS docs: https://git.k8s.io/community/contributors/guide/owners.md
+
+approvers:
+- ascerra
+- madunn
+- minlei98
+- ryankwilliams
```
Comment (ryankwilliams) [inline ci-operator/jobs/rh-openjdk/xpaas-openjdk/OWNERS:11]:
```suggestion
```

File: ci-operator/jobs/rh-openjdk/xpaas-openjdk/OWNERS [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+- vi-patel
+options: {}
+reviewers:
+- ascerra
+- madunn
+- minlei98
+- ryankwilliams
```
Comment (ryankwilliams) [inline ci-operator/jobs/rh-openjdk/xpaas-openjdk/OWNERS:18]:
```suggestion
```

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rh-openjdk-jdkContainerOcpTests-main-openjdk-ocp4.14-lp-interop-openjdk-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rh-openjdk-jdkContainerOcpTests-main-openjdk-ocp4.14-lp-interop-openjdk-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rh-openjdk-jdkContainerOcpTests-tar_uid_fix-openjdk-ocp4.14-lp-interop-openjdk-interop-aws

**Comment by smatula** [comment]
/pj-rehearse periodic-ci-rh-openjdk-jdkContainerOcpTests-main-openjdk-ocp4.14-lp-interop-openjdk-interop-aws

**Review by calebevans — APPROVED** [review]
/lgtm

**Pull request merged**